src/specfuzz_rtl.S

```diff
@@ -487,16 +494,59 @@ specfuzz_rlbk:
     je .L2
         movq (%rsp), %rbx
         movq 8(%rsp), %rcx
-        cmp %rbx, (%rcx)
-        jne .L1
-        addq $16, %rsp
+		movq 16(%rsp), %rdx
```

Comment:
Could you add a comment to explain what's going on here? It's very hard to read raw assembly without explanations.